.gitignore

```
@@ -1,6 +1,7 @@
-/coverage/
+/old/
```

Please add coverage to your .gitignore it keeps the pull request manageable in size. 

```suggestion
/old/
coverage
```